packages/ui/src/components/va-dropdown/VaDropdown.stories.ts

```diff
@@ -1,12 +1,386 @@
+            </va-dropdown-content>
+          </template>
+        </va-dropdown>
+      </VaValue>
+    `,
+})
+
+export const Cursor = () => ({
+  components: { VaDropdown },
+  template: `
+      <va-dropdown
+          trigger="right-click"
+          placement="bottom-start"
+          cursor
+      >
+        <template #anchor>
+          <div style="width: 8rem; height: 8rem; border: var(--border-dashed)">
+            Right click
+          </div>
+        </template>
+        Ctx dropdown
+      </va-dropdown>
+    `,
+})
+
+export const CursorWithTarget = () => ({
+  components: { VaDropdown },
+  setup () {
+    return {
+      cursorTarget: ref(null),
+    }
+  },
+  template: `
+      <va-dropdown
+          trigger="right-click"
+          placement="bottom-start"
+          :target="cursorTarget"
+          cursor
+      >
+        <template #anchor>
+          <div
+              style="width: 8rem; height: 8rem; border: var(--border-dashed)"
+              ref="cursorTarget"
+          >
+            Right click
+          </div>
+        </template>
+        Ctx dropdown
+      </va-dropdown>
+    `,
+})
+
+export const AnchorSelector = () => ({
+  components: { VaDropdown },
+  template: `
+      <div
+          style="border: var(--border-dashed)"
+          id="anchor-selector"
+      >
+        Click me
+      </div>
+      <va-dropdown
+          anchor-selector="#anchor-selector"
+      >
+        Anchor selector
+      </va-dropdown>
+    `,
+})
+
+export const InnerAnchorSelector = () => ({
   components: { VaDropdown },
-  template: '<VaDropdown/>',
+  template: `
+      <va-dropdown
+          inner-anchor-selector="#inner-anchor-selector"
+      >
+        <template #anchor>
+          <div style="border: var(--border-dashed)">
+            <div
+                style="margin: 1rem; border: var(--border-dashed)"
+                id="inner-anchor-selector"
+            >
+              Click me
+            </div>
+          </div>
+        </template>
+        Inner anchor selector
+      </va-dropdown>
+    `,
+})
+
+export const AnchorBoundProps = () => ({
+  components: { VaDropdown, VaIcon },
+  template: `
+      <va-dropdown>
+        <template #anchor="{ isOpened }">
+          <div>
+            <va-icon :name="isOpened ? 'va-arrow-up' : 'va-arrow-down'"/>
+          </div>
+        </template>
+        Clicked
+      </va-dropdown>
+    `,
+})
+
+export const KeepHeight = () => ({
+  components: { VaDropdown, VaButton, VaDropdownContent },
+  template: `
+      <div style="height: 1000px">
+        <va-dropdown>
+          <template #anchor>
+            <va-button>Anchor</va-button>
+          </template>
+          <va-dropdown-content>
+            <div style="height: 600px; width: 100px;">
+              Hi
+            </div>
+          </va-dropdown-content>
+        </va-dropdown>
+      </div>
+    `,
+})
+
+export const Autoplacement: StoryFn = () => ({
```

Comment:
Need an example for default autoplacement behavior.

It's not clear to me how using `autoplacementTargetRef` differs from not using it.

Reply:
Default autoplacement behaviour without target doesn't seem to handle overflow properly and dropdown gets positioned outside of scroll parent. Is this needed to be showcased? I thought that in this case target needs to be specified for everything to work correctly and I doubt that improper usage should be shown

Reply:
Or is current behaviour without specifying the target considered a bug?